[CI:DOCS] docs: make CNI removal explicit (#20578)
containers/podman: docs/source/markdown/podman-network-create.1.md

Upon completion of creating the network, Podman displays the name of the newly added network.

> NOTE: The support for the network name "pasta" is deprecated and will be removed in the next major

Reviewer: @Luap99 this might stick in my noggin better.  Sorry for the rehashing here, as EACK!, I think I suggested the current line in the first place.

```suggestion
NOTE: The support for "pasta" as a value passed in the *name* field is deprecated and will not be accepted in the next major
```

Author: Thanks, I reworded it a bit as I think using `field` is also confusing.

> release Podman 5.0 because it is used as a special network mode in **podman run/create --network**.

Reviewer: ```suggestion
Podman version 5.0, because it is used as a special network mode in **podman run/create --network**.
```

> The CNI backend is deprecated and will be removed in the next major Podman version 5.0 in preference

Reviewer: ```suggestion
The CNI backend is deprecated and will be removed in the next major Podman version 5.0, in preference
```

> The CNI backend is deprecated and will be removed in the next major Podman version 5.0 in preference

Reviewer: Soft suggestion
```suggestion
The CNI backend is also deprecated and will also be removed in the next major Podman version 5.0,  in preference
```
